sansio_lsp_client/client.py

```diff
@@ -285,3 +495,111 @@ def completions(
         if context is not None:
             params.update(context.dict())
         return self._send_request(method="textDocument/completion", params=params)
+
+    def hover(self, text_document_position: TextDocumentPosition) -> int:
+        assert self._state == ClientState.NORMAL
+        return self._send_request(
+            method="textDocument/hover", params=text_document_position.dict()
+        )
+
+    def signatureHelp(self, text_document_position: TextDocumentPosition) -> int:
+        assert self._state == ClientState.NORMAL
+        return self._send_request(
+            method="textDocument/signatureHelp", params=text_document_position.dict()
+        )
+
+    def definition(
+        self,
+        text_document_position: TextDocumentPosition,
+        # TODO PartialResultParams
```

Comment:
What does this comment mean?

Reply:
I don't know, but if it's something important, we'll discover it later. I deleted the comments for now. They really aren't of any value when nobody understands them.